README.md

```diff
@@ -1,37 +1,41 @@
 # FaunaDB
 
-VSCode extension for FaunaDB.
+VS Code extension for FaunaDB.
 
-## Features
+## Getting Started
 
-### Browse your database data
+To get started using the Fauna VS Code extension, you'll need to set your secret key in order to access database information. Keys can be created in the Fauna Console webapp or via the Shell CLI.
 
-You can browse databases, indexes, collections, documents and functions.
+To set the key, go to `Code > Preferences > Settings > Extensions > FaunaDB`.
 
-![Browser your database data](media/browse-feature.png)
+![Extension settings](media/extension-settings.png)
 
-### Run queries
+* `faunadb.secretKey`: Your database secret.
 
-![Run queries](media/query-feature.gif)
+> WARNING: Be careful! To avoid exposing this secret, do **not** commit your local `.vscode` configuration.
```

Comment:
Sometimes you want to commit the `.vscode` to attach some configurations to the project. Maybe could be better to specify just the secret config.

Reply:
@BrunoQuaresma Agreed. How about something like this?

```
WARNING: Be careful! To avoid exposing this secret, do not commit it to your local `.vscode` configuration.

Reply:
Looks great.